### ISSUE TITLE
Request for Site Monitoring fields for Odonata forms

### ISSUE THREAD

**Issue by DavidHepper** [BiologicalRecordsCentre/iRecord#1444]
We would like to add a Monitoring tab to the two current Dragonfly-specific forms (single and multi-species), please, for use in iRecord Desktop, iRecord App and spreadsheet imports. I'm guessing that this will take a bit of negotiation but I've set out our requests below. Where possible, I have borrowed fields from UKBMS surveys. I see BDeed is asking for something similar (via the Forum).

_Fields for the new Monitoring tab of the Form:_

Field Prompt | Field Type | Picklist Values / Format | Comments
--- | --- | --- | ---
Data collected as Monitoring? | Checkbox | + unchecked (default) + checked | Ideally, checking this box should unlock the other fields. Return 0 or 1.
Start time (hh:mm) | Time | hh:mm | Return text.
End time (hh:mm) | Time | hh:mm | ditto
Temp (°C) | Integer | 0 .. 40 |  
Wind speed | Integer picklist | 0 .. 6 (Beaufort) | With Beaufort standard descriptions, as UKBMS.
% Sun | Integer picklist | 0, 20, 40, .. 100 % | Or this field could be % Cloud cover (the inverse) instead.
Additional information | Multi-line Text |   |  
Sex | Text picklist | m, f, m+f (or leave blank) |  
Site photos | Media attachment |   | In addition to observation photos. Ideally, this field should be on the 'Where was it?' tab.
Improvements to Existing Fields: |   |  
Site type | Text picklist | + Lake (2 + ha, ha= 100m x100m) + Large pond (0.1-2ha, ha= 100m x100m) + Small pond (<0.1ha, ha= 100m x100m) + River (over 3m wide) + Stream (30cm-3m wide) + Runnel (less than 30cm) + Canal + Drain/ditch (manmade drainage channel) + Flush/spring (water from underground flows out to create an area of saturated ground) + Swamp/fen (wet area normally covered by water, with tall reeds and grass-like vegetation) + Upland bog (wet peatland above 300m altitude) + Lowland bog (wet peatland below 300m altitude) + Other (please state): | The current list needs greater clarity.Return only the main type, not including the explanatory comment.
Life Stage Abundance fields | Text picklist plus override | 1, 2-5, 6-20, etc. plus direct integer entry | Each field currently allows only standard ranges. Is it possible to give the option to enter a number such as 4 or 38?

**Comment by DavidHepper** [comment]
Further to this, I'm looking into automating the entry of meteorological data by web API calls to a colleague's site that retrieves and logs data from METAR. Is this something that could be run on a schedule by an iRecord background process, alongside assignment to VC perhaps? It would remove the onus on the observer to fill in all the survey meto data, could be more accurate than human recording and would certainly be more consistent across observers.

**Comment by DavidRoy** [comment]
We do this in the iRecord butterflies app

**Comment by DavidHepper** [comment]
Is there an ETA for this, please? I need to report back.

**Comment by DavidRoy** [comment]
I think it would help to discuss this via a separate meeting as there are several options

**Comment by DavidHepper** [comment]
Yes, please @DavidRoy - I'm available at various times over the next few days, if you'd like to propose some options.

**Comment by DavidHepper** [comment]
@DavidRoy we're into the dragonfly flight season. Although I'm keen for commonality with other Monitoring forms we really would like to make progress on this one way or another.
